debug.go

```diff
@@ -0,0 +1,171 @@
+package rossby
```

Comment:
This library is copied over from Alia/Raft/ePaxos etc -- I'd like to move this into our own kansaslabs logging package that we use across all of our Go systems. E.g. `github.com/kansaslabs/x` -- happy to discuss this more in the future.